xep-0171.xml

```diff
@@ -440,7 +440,7 @@
           <xs:attribute name='charset' type='xs:string' use='optional'/>
           <xs:attribute name='source_lang' type='xs:language' use='optional' />
           <xs:attribute name='destination_lang' type='xs:string' use='required'/>
```

Comment:
Actually the datatype for destination_lang needs to be corrected to 'xs:language', too.